ci/docs/pulp_index_pulp3.rst

```diff
@@ -42,4 +41,4 @@ Index and Search
 .. toctree::
    :maxdepth: 1
 
-   all_content_index_pulp3
+   all_content_index
```

Comment:
@werwty , yeah, that's a confusing part which I got wrong last time...
No matter what the filename in this for the all_content_index is, [it will be copied and become an `all_content_index.rst`](https://github.com/goosemania/pulp_packaging/blob/2aa8032bb18759d09a9a1e2b9e8aa82d3bbef3c6/ci/docs-builder.py#L97-L99). So `all_content_index` in a toctree is correct for any Pulp version.